### ISSUE TITLE
[css-values-5] Are the tree counting functions allowed inside calc()?

### ISSUE THREAD

**Issue by weinig** [w3c/csswg-drafts#10919]
The tree counting functions introduced in CSS Values 5, https://drafts.csswg.org/css-values-5/#tree-counting, don't state if they are usable in calc(), and don't declare themselves to be "math functions" like `random()` does for instance - https://drafts.csswg.org/css-values-5/#random.

Should they be usable from within calc()?

**Comment by Loirooriol** [comment]
They are still defined as `<integer>`s, which are accepted in `calc()`

**Comment by tabatkins** [comment]
Yeah, the grammar for calculations in math functions bottoms out in `<number>`, so anything that's that type is meant to work.

**Comment by weinig** [comment]
Thanks.
